### DISCUSSION TITLE
config help - updating a list of docker images

### DISCUSSION THREAD

**Discussion by stocksr** [renovatebot/renovate#27504] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### What is your question?

we have a file in our repository - called `image.lst` that contains a list of docker images that are used in other parts of our code base. (this is used to pull an offline copy of the images for an air-gapped install, but I don't think that is relevant here)

this file has one docker image/tag/digest per line  e.g. looks a bit like this (the actual file is a bit longer) 
```
busybox:1.36.1@sha256:6d9ac9237a84afe1516540f40a0fafdc86859b2141954b4d643af7066d598b74
curlimages/curl:8.6.0@sha256:c3b8bee303c6c6beed656cfc921218c529d65aa61114eb9e27c62047a1271b9b
leominov/redis_sentinel_exporter:1.7.1@sha256:812e617d2201d0ee8ba51d60b9fe9590a46bed84abc95746f554e1ca04e5478a
oliver006/redis_exporter:v1.57.0@sha256:5f2a7e12fa3d075f04da2175643e3c70695f6a58b4010d80f08b5f48400cbe1c
redis:7.2.4-alpine@sha256:995e6eaab954e92cadf1a0bb1eab71ae3baae8e02ea4354fd8aa136a61e42247
```
renovate is happily creating PR's to update our kubernetes manifests (yaml files) with new images, is there any way I can tell renovate about **this** file and get it to update the relevant line here at the **same time**?

I would be open to minor changes to the file format / name as well if that was needed to get this working, as long as I can still process the list of images in a script afterwards.

Many thanks

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Yes, use a custom (regex) manager: https://docs.renovatebot.com/modules/manager/regex/
